streaming-binlog/src/main/java/org/openmrs/analytics/DebeziumListener.java

```diff
@@ -84,11 +84,11 @@ FhirConverter createFhirConverter(CamelContext camelContext) throws IOException,
 	
 	private String getDebeziumConfig() throws IOException {
 		Map<String, String> debeziumConfigs = this.generalConfiguration.getDebeziumConfigurations();
-		return "debezium-mysql:" + debeziumConfigs.get("databaseHostName") + "?" + "databaseHostname="
+		return "debezium-mysql:" + debeziumConfigs.get("databaseServerName") + "?" + "databaseHostname="
 		        + debeziumConfigs.get("databaseHostName") + "&databaseServerId=" + debeziumConfigs.get("databaseServerId")
 		        + "&databasePort=" + debeziumConfigs.get("databasePort") + "&databaseUser="
 		        + debeziumConfigs.get("databaseUser") + "&databasePassword=" + debeziumConfigs.get("databasePassword")
-		        + "&databaseServerName=" + debeziumConfigs.get("databaseName") + "&databaseWhitelist="
```

Comment:
FYI, there is at least one reference to `--databaseName` in README but it is in the section that you have not changed yet (OpenHIM). So I suppose you will fix it when taking that section out of the main README.